autogpt_platform/frontend/src/app/page.tsx

```diff
@@ -87,9 +87,7 @@ const Monitor = () => {
         selectedFlow={selectedFlow}
         onSelectFlow={(f) => {
           setSelectedRun(null);
-          setSelectedFlow(
-            f.id == selectedFlow?.id ? null : (f as GraphMetaWithRuns),
-          );
+          setSelectedFlow(f.id == selectedFlow?.id ? null : (f as GraphMeta));
```

Comment:
Maybe outside the scope but it'd be good to rename all `*flow` to `*graph` and `*run` to `*execution` for consistency.